docker-compose.yml

```diff
@@ -13,15 +13,15 @@ services:
       context: .
       # Use target devcontainer-trt for TensorRT dev
       target: devcontainer
-    deploy:
-          resources:
-              reservations:
-                  devices:
-                      - driver: nvidia
-                        count: 1
-                        capabilities: [gpu]
-    devices:
-      - /dev/bus/usb:/dev/bus/usb
+    # deploy:
+    #       resources:
+    #           reservations:
+    #               devices:
+    #                   - driver: nvidia
+    #                     count: 1
+    #                     capabilities: [gpu]
+    # devices:
+    #  - /dev/bus/usb:/dev/bus/usb
```

Comment:
I think more often than not nvidia isn't going to be used here, but we should have a comment explaining when this would need to be enabled

Reply:
my apologies! this file was NOT supposed to be committed.

Reply:
@NickM-27 restored the original yaml